salt/_modules/metalk8s_solutions.py

```diff
@@ -324,6 +324,11 @@ def manifest_from_iso(path):
             )
         )
 
+    if manifest is None:
```

Comment:
Does it mean `result['stdout']` is an empty string? If that's the case, can't we tune the `isoinfo` command to make sure it breaks on a missing file?

Reply:
Doesn't seem to be achievable, at least with the current version (1.1.11) we ship.

Reply:
Or we could do it with an ugly thing like:
```
isoinfo -x 'MANIFEST.YAML;1' -f </path/to/the/solution.iso> | grep ''
```
grep will fail because there is nothing to read.

Reply:
OK thanks for checking :) sorry I made you, the point was more that it's weird to say the file doesn't exist after having tried to parse its contents as YAML, but it's really minor

Reply:
Oh ok, I've moved the test before the YAML parsing.